src/OrchardCore/OrchardCore.ContentManagement.GraphQL/Options/GraphQLContentOptions.cs

```diff
@@ -106,7 +106,7 @@ public bool IsCollapsedByDefault(ContentTypePartDefinition definition)
                 return true;
             }
 
-            var contentTypeOption = ContentTypeOptions.FirstOrDefault(ctp => ctp.ContentType == contentType);
+            var contentTypeOption = ContentTypeOptions.FirstOrDefault(ctp => String.Equals(ctp.ContentType, contentType, StringComparison.OrdinalIgnoreCase));
```

Comment:
Why are you adding so often `StringComparison.OrdinalIgnoreCase`, here better to keep `ctp.ContentType == contentType` unless it really fixes an issue.

Reply:
@jtkech retrieving content types/part is case insensitive. And in this case, one can configure the content type with case insensitive. I can change it back if needed

Reply:
Oh yes, you did it recently for lookup/searching in some places, right? But here I think options are populated with `contentType` coming from existing `ContentTypePartDefinition`.

Reply:
Yes that was added recently. In here not it does not come from the definition. There is a method `ConfigureContentType(...)` that configure the content type is a free form which is why I added it here

Reply:
Ah okay, cool then